src/open_clip/modified_resnet.py

```diff
@@ -0,0 +1,172 @@
+from collections import OrderedDict
```

Comment:
Is there any diff compared to code in main ?

Reply:
updated now so all is copied from main